app/import/licence/services/fetch-licence-from-import.service.js

```diff
@@ -0,0 +1,96 @@
+'use strict'
+
+/**
+ * Service for /import/licence
+ * @module ImportLicenceService
+ */
+const { db } = require('../../../../db/db.js')
+
+async function go (licenceRef) {
+  const licence = await _getLicenceByRef(licenceRef)
+
+  const { ID: id, FGAC_REGION_CODE: regionCode } = licence
+
+  // TODO: rename these ? why section 130 ?
+  const [tptAgreements, section130Agreements] = await Promise.all([
+    _getTwoPartTariffAgreements(regionCode, id),
+    _getSection130Agreements(regionCode, id)
+  ])
+
+  return {
+    licence,
+    section130Agreements,
+    tptAgreements
+  }
+}
+
+async function _getTwoPartTariffAgreements (regionCode, licenceId) {
+  const query =
+    `  SELECT a.*,
```

Comment:
The sql used to get the data from the import table can and should be used with minmal changes. 

Below is an example.